docs/content/concepts/logging/loggers.mdx

```diff
@@ -193,6 +193,31 @@ width={2048}
 height={804}
 />
 
+### Specifying Default Loggers
```

Comment:
I think it's worth mentioning this is at the repository level in the heading.

Reply:
```suggestion
### Specifying Default Loggers on Repository
```